doc/content/guides/connecting-gateways/multitechconduit/_index.md

```diff
@@ -36,20 +36,19 @@ Click on **LoRaWAN<sup>®</sup>** in the menu on the left. It opens the Gateway&
 {{< figure src="005_Gateway_Menu_LoRaWAN.png" alt="MultiTech Conduit gateway home page" >}}
 
 Under **Network Settings**, select the mode as &quot;Packet Forwarder&quot;.
+
 {{< figure src="006_Packet_Forwarder_Gateway.png" alt="MultiTech Conduit packet forwarder settings" >}}
 
-Select your Channel Plan (EU868 in this case).
+Select the applicable Channel Plan from the dropdown (EU868 in this case).
```

Comment:
Is there a way to customize the channel plan, i.e. are these stored in a folder somewhere with an index?

The thing is that we need to be able to customize these. In fact we have a very nice endpoint that generates a `global_conf.json` compatible JSON file with the actual channel plan of the gateway.

EU868 happens to be the same as what we use in TTN, but this is not a general rule, especially not outside Europe. I'll check in with MultiTech to see what we can do here.

Reply:
@johanstokking 
1. The Channel plans are indeed a dropdown. ![Multitech_Conduit_Settings](https://user-images.githubusercontent.com/57247204/73279655-77351200-4213-11ea-96ce-d4541429e9ca.png)

2. The Multitech documentation has references to the json file to make the changes: https://www.multitech.net/developer/software/lora/conduit-mlinux-convert-to-basic-packet-forwarder/

3. The file(s) are to be stored at `/var/config/lora/` in the gateway filesystem. 

4. A sample config file for EU868, Conduit AP:  http://git.multitech.net/cgi-bin/cgit.cgi/meta-mlinux.git/plain/recipes-connectivity/lora/lora-packet-forwarder/global_conf.json.3.1.0.MTCAP-LORA-1-5.EU868.basic?h=3

5. The index where all such files are hosted: http://git.multitech.net/cgi-bin/cgit.cgi/meta-mlinux.git/plain/recipes-connectivity/lora/lora-packet-forwarder/

Reply:
OK I see.

But can we load custom frequency plans to that folder and have them shown up here?

@adriansmares can you comment on how to generate the JSON file via GCS (and potentially get that documented)?

Reply:
This has already been used by the `NASys LoRaWAN Outdoor Gateway` connection guide, see [the section on configuration using a terminal](https://github.com/TheThingsNetwork/lorawan-stack/blob/master/doc/content/guides/connecting-gateways/nasysoutdoorgateway/_index.md).

Reply:
@abhinavdgupt if we can add instructions to update these frequency plans like mentioned above, and still use the dropdown, that would be best